licenses/views.py

```diff
@@ -46,16 +46,31 @@ def home(request):
         .distinct("language_code")
         .values_list("language_code", flat=True)
     )
+    codes_for_10 = (
+        License.objects.filter(version="1.0")
+        .order_by("license_code")
+        .distinct("license_code")
+        .values_list("license_code", flat=True)
+    )
+    languages_for_10 = (
+        LegalCode.objects.filter(license__version="1.0")
+        .order_by("language_code")
+        .distinct("language_code")
+        .values_list("language_code", flat=True)
+    )
 
     licenses_by_version = [
         ("4.0", codes_for_40, languages_for_40),
+        ("1.0", codes_for_10, languages_for_10),
     ]
+    print(codes_for_40)
+    print(codes_for_10)
```

Comment:
should we remove this?

Reply:
yes, more left-over debugging code, sorry.